api/src/db/seeder.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose')
```

Comment:
Good practice when you create new branches to branch off of the master branch, and not your previous branch, otherwise the "seed" code gets mixed in with the account create code. Do you know to rebase to fix this? (If you are not good with rebasing make sure to copy your files first otherwise you may lose them)